ingestor/cluster/replicator.go

```diff
@@ -83,8 +83,8 @@ func NewReplicator(opts ReplicatorOpts) (Replicator, error) {
 
 func (r *replicator) Open(ctx context.Context) error {
 	ctx, r.closeFn = context.WithCancel(ctx)
-	r.wg.Add(5)
-	for i := 0; i < 5; i++ {
+	r.wg.Add(50)
```

Comment:
Going to make this configurable as part of this PR.  This change is actually dangerous for collector in large deployments, but good for ingestor.